### ISSUE TITLE
Error when rendering Single Message view for message in deleted input

### ISSUE THREAD

**Issue by danotorrey** [Graylog2/graylog2-server#6174]
An error occurs when the following GET request is executed when viewing the Single Message page for a message that was received through an input which is now deleted. 

`http://localhost:9000/api/system/inputs/{input-id}`
![image](https://user-images.githubusercontent.com/3423655/61565881-08932280-aa40-11e9-9040-5867f7c72cc5.png)

## Expected Behavior
I am not sure what is loading this Input details resource on this page, but it does not work when the input is deleted :)

## Steps to Reproduce (for bugs)
1. Receive a message in Graylog.
2. Delete the input that the message was received through.
3. Go to the search page and find the message.
4. Click the message to go to the Message Details page. 

## Your Environment

* Graylog Version: Graylog 3.1.0-beta.3-SNAPSHOT (master)

**Comment by danotorrey** [comment]
Found when reviewing https://github.com/Graylog2/graylog2-server/pull/6160

**Comment by edmundoa** [comment]
@danotorrey could you please check, if this is still relevant? Thank you!

**Comment by danotorrey** [comment]
@edmundoa Certainly! Just retested on latest `master` and I can still reproduce the issue. It's not that big of a problem, because other than the error, nothing on the Message Details seems to be broken.
